caravel/models.py

```diff
@@ -601,6 +584,24 @@ def sql_url(self):
     def sql_link(self):
         return '<a href="{}">SQL</a>'.format(self.sql_url)
 
+    def dttm_converter(self, dttm, tf=None, db_expr=None):
```

Comment:
`self` isn't used in this method, but both `tf` and `db_expr` live in the column object. Maybe this method should be a method of the column itself, and receive only `dttm`.

What about renaming the method `dttm_sql_literal`?